tests/index.ts

```diff
@@ -68,7 +68,7 @@ root.requestAnimationFrame = () => { };
 
 root.MediaStream = () => { };
 
-root.navigator = {
+root.navigator ??= {
```

Comment:
Can you provide an explanation why this is needed? It's great if you have a link/source.

Reply:
Yes, looks like [globalThis.navigator](https://github.com/nodejs/node/issues/39540#issuecomment-1770612761) added in Node v22. Hence, the error `Cannot set property navigator of #<object> which has only a getter`.

Reply:
If I remove lines 71 - 83, the tests doesn't even fail. Does this mean we don't need this anymore?

Reply:
@charliesantos yes, but it will fail on Node v20, and I though we still want tests to pass on v20 since we use it for "stable" docker containers.

Reply:
Good call. In this case, let's add a comment here to prevent confusion
>// This only applies to Node v21 and earlier, as v22 already includes globalThis.navigator.